src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -7,7 +7,7 @@
 
 # pylint: disable=line-too-long
 
-image_long_summary = """                      URN aliases: CentOS, CoreOS, Debian, openSUSE, RHEL, SLES, UbuntuLTS, Win2008SP1, Win2012Datacenter, Win2012R2Datacenter.
+image_long_summary = """                      URN aliases: CentOS, CoreOS, Debian, openSUSE, RHEL, SLES, UbuntuLTS, Win2008R2SP1, Win2012Datacenter, Win2012R2Datacenter.
```

Comment:
Please ignore my comments. The link appears working now.
It didn't work yesterday and I was redirected to a general azure landing page.